library/VTKExtensions/Rendering/vtkF3DPolyDataMapper.cxx

```diff
@@ -130,25 +132,64 @@ void vtkF3DPolyDataMapper::ReplaceShaderValues(
   type = uniforms->GetUniformTupleType("jointMatrices");
   if (type != vtkUniforms::TupleTypeInvalid)
```

Comment:
currently a VTK 9.0 user will just not get a deformed mesh without any kind of warning. It would be nice to fix if not too hard.

Reply:
I don't see how to do that. If there's no custom uniform, it could mean there is no animation or we are using 9.0. 
I'm not sure it's possible to know that in f3d

Reply:
right